platform-test/base/unification.oz

```diff
@@ -0,0 +1,106 @@
+functor
+import
+   System(show:Show)
+export
+   Return
+define
+   ShouldHaveFailed = 'should have failed'
+   proc{MustFail X P}
+      skip
+      try
+         {P}
+         raise ShouldHaveFailed end
+      catch S then case S of !ShouldHaveFailed then
+         raise '  '(X ShouldHaveFailed) end
```

Comment:
Because the testing script does not print strings nicely and I wanted to be able to discriminate between errors succintly, this is one way I found. Trying to get it cleaner though!